### DISCUSSION TITLE
Errors in last changes of template "with-tailwind"

### DISCUSSION THREAD

**Discussion by madmonkey08** [nandorojo/solito#485] [Q&A]
Hey guys, I'm having some troubles with last changes in the "with-tailwind" template, specifically with bun's update. First of all, when I try to run NextJS application, it fails because autoprefixer isn't found, solving this error was easy, I only have to install autoprefixer, but there is another error, now related to expo that I can't see where it is, expo app doesn't find the default features in packages/app, I just execute the command:

npx create-solito-app@latest my-solito-app -t with-tailwind

So that's the initial configuration, it shouldn't fail, I've installed dependencies with bun install. Thank you all for your attention.

**Comment by nandorojo** [comment]
#480 may be the cause.

@dnlgrgly any ideas?

**Reply by danger9224** [reply]
Oof. I’ll look into this ASAP, no ideas off the top of my head. I didn’t have to install autoprefixer.

Sorry about the hiccup!
